### ISSUE TITLE
ABRT deletes core dumps far too aggressively, even while user is trying to report the crash

### ISSUE THREAD

**Issue by mcatanzaro** [abrt/abrt#1450]
I reported this downstream several years ago, but the issue is still valid:

https://bugzilla.redhat.com/show_bug.cgi?id=1047550

Looks like ABRT is deleting old core dumps before saving a new one. It should at least not delete one that it's currently reporting a bug against.

**Comment by ernestask** [comment]
Ideally, I think it would be nice if we didn’t copy core dumps at all, since we rely entirely on systemd-coredump.

Semi-related to https://github.com/abrt/abrt/issues/832, if systemd-coredump grows support for debuginfod.

**Comment by mcatanzaro** [comment]
Just hit this again.

I started reporting the Geary crash I noticed in https://github.com/abrt/abrt/issues/1455#issuecomment-593620177. 13 or 14 minutes later, ABRT finally told me that the retrace had failed. Instead of prompting me to retrace locally, it instead popped up two warning dialogs complaining that the problem directory no longer exists. ABRT had already deleted it sometime during that time window.
